src/main/java/jp/skypencil/errorprone/slf4j/LoggerMatcher.java

```diff
@@ -0,0 +1,19 @@
+package jp.skypencil.errorprone.slf4j;
+
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.matchers.Matcher;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.VariableTree;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use classes from the Java API instead of Sun classes. ([squid:S1191](https://sonarcloud.io/organizations/kengotoda-github/rules#rule_key=squid%3AS1191))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=jp.skypencil.errorprone.slf4j%3Aerrorprone-slf4j&pullRequest=4&issues=AWRd1zxg1IJZDfsS72jg&open=AWRd1zxg1IJZDfsS72jg)

[ISSUE_KEY]: AWRd1zxg1IJZDfsS72jg